uwblueprint/starter-code-v2
# Modify frontend to store current user in AuthContext

## Notion ticket link

[Update AuthContext to store full user info](https://www.notion.so/uwblueprintexecs/Update-AuthContext-to-store-full-user-info-db347002b5ff4c90afc89d13850f23dc)



## Implementation description
* Store user info returned on login in localstorage
* Removed the user_id and email placeholders and replaced with current authenticated user's user_id and email
* New util functions for getting and setting data from/to local storage
* Some frontend cleanup



## Steps to test
1. Run the application (should work either backends), go to the frontend http://localhost:5000
2. Login, test create entity, update entity, display entities



## What should reviewers focus on?
* Did anything break?


## Checklist
